### PR TITLE
Core ML examples in Drawing Classifier User Guide

### PR DESCRIPTION
**Pull request by syoutsey**
Add relevant Core ML integration Swift code to Drawing Classifier user guide.

### CODE REVIEW

File: userguide/drawing_classifier/export-coreml.md [hunk 1]
```diff
@@ -34,6 +34,18 @@ let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
 let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
     self?.processClassifications(for: request, error: error)
 })
+let grayscaleImage = UIImage(named: "MyImage")
+        
+let handler = VNImageRequestHandler(cgImage: grayscaleImage.cgImage!, options: [:])
```
Comment (TobyRoseman) [inline userguide/drawing_classifier/export-coreml.md]: I get the error: `Value of optional type 'UIImage?' must be unwrapped to refer to member 'cgImage' of wrapped base type 'UIImage'`

File: userguide/drawing_classifier/export-coreml.md [hunk 2]
```diff
@@ -34,6 +34,18 @@ let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
 let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
     self?.processClassifications(for: request, error: error)
 })
+let grayscaleImage = UIImage(named: "MyImage")
```
Comment (TobyRoseman) [inline userguide/drawing_classifier/export-coreml.md]: How do users create this given a file path?

**Comment by nickjong** [comment]
Let's address both halves of #1964 in one PR, since the part about passing the image to the request handler is the same in both the bitmap and stroke cases. I believe Shreya is working on a PR to handle #1964

**Comment by TobyRoseman** [comment]
Closing in favor of #1969

**Pull request closed without merging**